IfExcercises.java

```diff
@@ -0,0 +1,71 @@
+import java.util.Scanner;
+
+public class IfExcercises {
+	public static void main(String[] args) {
+		/*Napisz program, który sprawdza dla trzech boków trójkąta a ,
+		b i c wprowadzonych z klawiatury, czy tworzą one trójkąt pro-
+		stokątny (zakładamy, że a > 0 , b > 0 , c > 0 ).
+		*/
+
+		Scanner scanner = new Scanner(System.in);
+
+		int a, b, c;
+		System.out.println("Enter length sides triangle:" + "\n" + "Side a: ");
+		a = scanner.nextInt();
+		
+		System.out.println("Side b:");
+		b = scanner.nextInt();
+		
+		System.out.println("Side c");
+		c= scanner.nextInt();
+		scanner.nextLine();
+
+		if((a*a) + (b*b) == c*c)
```

Comment:
spaces between operators, add space after if, remove redundant braces
Multiplication will be calculated before summing so no need to use braces.
check https://introcs.cs.princeton.edu/java/11precedence/